pages/api/hostedPagesLink/[hostedPagesLinkId].ts

```diff
@@ -0,0 +1,67 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import jwt from 'jsonwebtoken'
+import { type HostedPagesLinkParams, environment } from '../../../types'
+
+type Data =
+  | {
+      stakeholderId: string
+      hostedPagesLinkId: string
+      pathwayId: string
+    }
+  | {
+      error: any
+    }
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  const { hostedPagesLinkId } = req.query as HostedPagesLinkParams
+
+  const token = jwt.sign(
+    {
+      username: environment.apiGatewayConsumerName,
+      feature: 'hosted-pages-link',
+      hosted_pages_link_id: hostedPagesLinkId,
```

Comment:
as I mentioned in the orchestration review, I would put this link in the subject